lib/hexpm_web/templates/page/index.html.eex

```diff
@@ -57,6 +57,22 @@
         </div>
       </div>
     </div>
+
+    <div class="using-with-box gleam">
+      <div class="shadow"></div>
+      <div class="text-content clearfix">
+        <div class="side-box">
+          <div class="lang-symbol"></div>
+        </div>
+        <div class="instructions">
+          <h3><a href="<%= ~p"/docs/rebar3-usage" %>">Using with Gleam</a></h3>
+          <p>
+            Add the Hex dependency to your <code>gleam.toml</code> file using the <code>lustre = "~> 5.2"</code> format.
```

Comment:
We don't use the `~>` syntax as it was a common source of confusion for both new BEAM folks and established Gleam folks.
```suggestion
            Add the Hex dependency to your <code>gleam.toml</code> file using the <code>lustre = ">= 5.2.0 and < 6.0.0"</code> format.
```